src/schemas/xspec.rnc

```diff
@@ -126,6 +138,7 @@ context =
 	## are applied to it, and any parameters included in the apply templates.
 	element context { 
 		common-attributes,
+		##The mode in which templates are applied to it, and any parameters included in the apply templates.
```

Comment:
As described in https://github.com/xspec/xspec/pull/115#issuecomment-292457562, please add a space after ```##``` to make it consistent with the rest of the file, i.e.: 
```
## The mode in which templates are applied to it, and any parameters included in the apply templates.
```